src/torchcodec/decoders/_video_decoder.py

```diff
@@ -151,13 +149,6 @@ def __init__(
             # TODONVDEC P2 rename 'default' into 'ffmpeg' everywhere.
             device_variant = "default"
 
-        # Legacy support for device="cuda:0:beta" syntax
-        # TODONVDEC P2: remove support for this everywhere. This will require
-        # updating our tests.
-        if device == "cuda:0:beta":
-            device = "cuda:0"
```

Comment:
This is the actual removal. Everything else in this PR is a consequence of that change - mostly just changing how we use the BETA backend in existing tests.